src/main/java/com/wavefront/config/GeneratorConfig.java

```diff
@@ -41,6 +41,10 @@
       "auto-generation.")
   private Integer traceTypesCount = 0;
 
+  @Parameter(names = {"--totalTraceCount"}, description = "Total number of traces for " +
+      "auto-generation. This parameter disables duration")
```

Comment:
the same here, the description needs to be improved

Reply:
Could you please mention what you prefer to be improved?

Reply:
Removed "auto" prefix